### DISCUSSION TITLE
Can Vector's Loki `sink` accept Docker Container name as `endpoint`, instead of HTTP url?

### DISCUSSION THREAD

**Discussion by phoenisx** [vectordotdev/vector#19991] [Q&A]
Is it possible to set sink type: loki to use docker container/service names as endpoints?
Similar to how we use container names in `exclude_containers` for Docker logs source?

> I have never used `Vector` log collector before, thus if the question I'm asking here seems to obvious please ignore 🙇🏽 .

**Comment by jszwedko** [comment]
I'm not sure if I understand the question exactly, but if you are asking if you can use the container name of a Loki container as the endpoint for the Vector `loki` sink to send data to that really depends on container networking outside of Vector itself. Vector will just try to resolve the domain name in the endpoint using the normal DNS lookup mechanisms.

**Reply by phoenisx** [reply]
Thanks!
I'm trying to set up vector for the first time.

Will update if things work out.
